src/components/icons/GreenNimiqLogoOutlineWithStars.vue

```diff
@@ -0,0 +1,8 @@
+<template>
```

Comment:
Not sure if the name of file is great 🙃

Reply:
The color `#21BCA5` is the green color of the logo, should I create a CSS variable for that or hardcode it?

Reply:
That is `var(--nimiq-green)`.